wildfly-getting-started-archetype/src/main/resources/archetype-resources/src/test/java/org/wildfly/security/examples/test/SampleIT.java

```diff
@@ -0,0 +1,57 @@
+#set( $symbol_pound = '#' )
+#set( $symbol_dollar = '$' )
+#set( $symbol_escape = '\' )
+package org.wildfly.security.examples.test;
+
+import java.io.File;
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.importer.ZipImporter;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Sample integration test: demonstrates how to create the WAR file using the ShrinkWrap API.
+ * 
+ * Delete this file if no integration test is required.
+ */
+@RunWith(Arquillian.class)
```

Comment:
@darranl I would reconsider the testing strategy for this archetype.

As you use the `wildlfy-maven-plugin` to provision WildFly and deploy the deployment in it, you have a fully function runtime (server + deployments) that you can run your integration tests against.

We could focus our integration testing by running remote tests against a managed server that is at `target/server`.
This remove any need to use Shrinkwrap to "fake" the application and make sure that we test against the actual server that is serving our application.

Reply:
I opened https://github.com/darranl/wildfly-archetypes/pull/1 to change the testing strategy